container/test-app/index.html

```diff
@@ -601,7 +603,42 @@
           console.log('anhor', event.detail);
         });
 
-        //...
+        // uxManager(). closeUserSettings openUserSettings collapseLeftSideNav
+        // setDocumentTitle removeBackdrop hideAppLoadingIndicator
+        luigiContainer.addEventListener(MFEventID.CLOSE_USER_SETTINGS_REQUEST, event => {
```

Comment:
Integration tests are left out on this PR.
Need to restructure the Integration Test Environment because the index.html are getting bigger. #3562